Sming/system/include/esp_systemapi.h

```diff
@@ -32,6 +32,9 @@
  */
 #define SMING_UNUSED  __attribute__((unused))
 
+/* Flags a compiler warning when Sming framework methods, functions or types are changed */
+#define __deprecated __attribute__((deprecated))
```

Comment:
The other definitions of attributes are uppercase. Can you make also this one to be __DEPRECATED or SMING_DEPRECATED ?

Reply:
I like SMING_DEPRECATED, it's specific